fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -978,7 +990,7 @@ public void enableVersioning() {
 
   @Override
   public void disableVersioning() {
-      LOGGER.warn("Review if method (disableVersioning) can be removed after implementing Memento!");
+    getDescription().getChild(LDPCV_TIME_MAP).delete();
```

Comment:
Actually, this is a situation where `FedoraResourceImpl.getTimeMap()` could be used.